src/test/components/components-basic.test.ts

```
@@ -0,0 +1,82 @@
+    expect(typeof mockProps.slug).toBe("string");
+  });
+
+  it("devrait valider la logique de troncature pour ArticleCard", () => {
+    // Test de la logique métier sans rendu Astro
+    const longDescription = "Cette description est très longue et devrait être tronquée automatiquement pour s'adapter au format compact des cartes d'articles dans la grille principale de la section.";
+    const shortDescription = "Description courte";
+
+    // Reproduire la logique de troncature du composant
+    const truncateLongDescription = longDescription.length > 120 
+      ? longDescription.substring(0, 120) + "..." 
+      : longDescription;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)